src/Conductor.Domain/Services/ConductorRegistry.cs

```diff
@@ -20,15 +20,14 @@ public WorkflowDefinition GetDefinition(string workflowId, int? version = null)
         {
             if (version.HasValue)
             {
-                var entry = _registry.FirstOrDefault(x => x.Item1 == workflowId && x.Item2 == version.Value);
-                // TODO: What in the heck does Item3 mean?
-                return entry?.Item3;
+                var entry = _registry.FirstOrDefault(x => x.WorkflowId == workflowId && x.Version == version.Value);
+                return entry.WorkflowDefinition;
```

Comment:
removed null propgation as it's now a value tuple, but the default value of WorkflowDefinition will be null so this returns the same as propogating null :)